src/Microsoft.TemplateEngine.Cli/TemplatePackageCoordinator.cs

```diff
@@ -65,6 +66,54 @@ internal class TemplatePackageCoordinator
             return (await managedTemplatePackage.ManagedProvider.GetLatestVersionsAsync(new[] { managedTemplatePackage }, cancellationToken).ConfigureAwait(false)).Single();
         }
 
+        internal async Task<(string, string)> ValidatePackageAvailabilityAsync(ITemplateInfo template, CancellationToken cancellationToken)
+        {
+            _ = template ?? throw new ArgumentNullException(nameof(template));
```

Comment:
How about replacing this with [`[NotNull]`](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.notnullattribute)?

Reply:
I think it's no longer needed as whole assembly is nullable now.